pkg/kudoctl/env/environoment.go

```diff
@@ -62,3 +62,12 @@ func setFlagFromEnv(name, envar string, fs *pflag.FlagSet) {
 		}
 	}
 }
+
+// FlagNotChangedValue used for deviations from global defaults
+func (s *Settings) FlagNotChangedValue(fs *pflag.FlagSet, name, value string) string {
```

Comment:
could we find a better name for this? It is not really saying anything that it returns a Namespace so it could be very confusing... also I would go with more verbose docs for this one... 

Also value should probably be called more specific, like `overrideDefaultNamespace`

Reply:
yeah... I struggled with the naming :)   I wanted to call it `Or` or `FlagValueOr`.   
I did at one time have "namespace" in the fn name and decided to make it more generic so `OverrideDefault` was one I was considering and you came up with your name independently.

Reply:
also... there isn't a great way to handle the default being default without the lie :(

Reply:
yeah I did not think of anything either, that's why I was kind of leaning to moving out of global flags to local flags, I have a rough idea in my head how to make that in a ok-ish way. I might try to go that way next week when I am on the train. That looks like a good train exploratory task :)